src/Pages/Overview/OverviewProfile.js

```diff
@@ -17,11 +17,13 @@ export default function OverviewProfile (props) {
   return (
     <tr>
       <td>
-        {props.user.firstName[0].toUpperCase() +
-          props.user.firstName.slice(1, props.user.firstName.length) +
-          ' ' +
-          props.user.lastName[0].toUpperCase() +
-          props.user.lastName.slice(1, props.user.lastName.length)}
+        <div className='name'>
+          {props.user.firstName[0].toUpperCase() +
```

Comment:
can you move this to a function in [`src/APIFunctions/Profile.js`](https://github.com/SCE-Development/Core-v4/blob/dev/src/APIFunctions)?

I ask because the same logic for first and last name is done [here in `src/Pages/Profile/MemberView/Profile.jsx`](https://github.com/SCE-Development/Core-v4/blob/05287caeb4f09ad7487212ea6f529261364c5a87/src/Pages/Profile/MemberView/Profile.jsx#L25) too. 

You can define a funciton in the file like this

```js
export function formatFirstAndLastName(user) {
  return user.firstName[0].toUpperCase() +
            user.firstName.slice(1, user.firstName.length) +
            ' ' +
            user.lastName[0].toUpperCase() +
            user.lastName.slice(1, user.lastName.length)
```

Now, you can call on this helper by importing it at the top and the line here becomes
```jsx
{formatFirstAndLastName(props.user)}
```

Reply:
The function that you defined doesn't actually format the first and last name, so should I call it getFirstAndLastName() instead?

Reply:
You would send in a parameter `user`, which your function does the work of capitalizing the first letters of the name.